apps/api/src/app/tenant/tenant.controller.ts

```diff
@@ -106,4 +111,25 @@ export class TenantController {
       })
     );
   }
+
+  @Delete('/:identifier')
+  @ExternalApiAccessible()
+  @UseGuards(JwtAuthGuard)
+  @ApiResponse(DeleteTenantResponseDto)
+  @ApiOperation({
+    summary: 'Delete tenant',
+    description: 'Deletes a tenant entity from the Novu platform',
+  })
+  async removeTenant(
```

Comment:
Let's be more consistent and make this endpoint similar to what we have in the `topics` controller, I mean the response and the status.

Reply:
thanks love the suggestion, will add it for sure.

Reply:
you are still returning the response but the status is "no content" ;)

Reply:
actually the `@HttpCode(HttpStatus.NO_CONTENT)` do the heavy lifting and removed the data from the response but there were definitely leftovers on the flow, remove the unnecessary remaining code.